arch/arm/src/stm32/stm32_sdio.c

```diff
@@ -1049,10 +1069,10 @@ static void stm32_datadisable(void)
 {
   uint32_t regval;
 
-  /* Disable the data path */
+  /* Disable the data path  */
 
-  putreg32(SDIO_DTIMER_DATATIMEOUT, STM32_SDIO_DTIMER); /* Reset DTIMER */
-  putreg32(0,                       STM32_SDIO_DLEN);   /* Reset DLEN */
+  putreg32(~0, STM32_SDIO_DTIMER); /* Reset DTIMER */
```

Comment:
```suggestion
  putreg32(UINT32_MAX, STM32_SDIO_DTIMER); /* Reset DTIMER */
```
nit: just a suggestion hehe

Reply:
I have recently grown found of ~0.  :)